pkg/gktest/read_constraints.go

```diff
@@ -74,20 +115,23 @@ func readTemplate(f fs.FS, path string) (*templates.ConstraintTemplate, error) {
 
 	v, isVersionless := t.(versionless)
 	if !isVersionless {
-		return nil, fmt.Errorf("%w: %T", nil, t)
+		// This likely means a
```

Comment:
nit: this comment is not complete

Reply:
Removed since it isn't clear what to say here.